apps/cloudinary2/src/locations/Field/Thumbnail.tsx

```diff
@@ -76,7 +123,7 @@ function getUrlFromAsset(installationParams: AppInstallationParameters, asset: C
     api_key: installationParams.apiKey,
   });
 
-  const transformations = `${asset.raw_transformation ?? ''}/c_fill,h_100,w_150`;
+  const transformations = `${asset.raw_transformation ?? ''}/c_crop,h_149,w_194`;
```

Comment:
I think that this could be tweaked a bit more, as it's hard to tell what some of the sample images are in the Thumbnail. For example, here is the same image in the original app vs. these new changes:

![Screenshot 2023-08-15 at 9 02 58 AM](https://github.com/contentful/marketplace-partner-apps/assets/62958907/c6e930e4-7c1c-4b8f-a154-7483419d0ed2)

![Screenshot 2023-08-15 at 9 02 51 AM](https://github.com/contentful/marketplace-partner-apps/assets/62958907/457a66cb-a8df-4191-98c4-065b9b000af5)

Reply:
Oh, yes that looks wrong.
Plan was to scale down and only crop one dimension. Basically css' `object-fit: cover`